_sass/_base.scss

```diff
@@ -318,13 +318,29 @@ footer.sticky-bottom {
   border-bottom: 1px solid var(--global-divider-color);
   text-align: center;
   padding-top: 2rem;
-  padding-bottom: 5rem;
+  padding-bottom: 3rem;
   h1 {
     color: var(--global-theme-color);
     font-size: 5rem;
   }
 }
 
+.tag-list {
```

Comment:
if you change the window size, the tag list does not behave responsively. could you make it responsive?